pkg/controller/provider/web/client.go

```diff
@@ -11,23 +11,20 @@ import (
 	"path"
 )
 
+//
+// Errors.
 var (
 	ProviderNotSupportedErr = errors.New("provider (type) not supported")
 	ProviderNotReadyErr     = errors.New("provider API not ready")
+	RefNotUniqueErr         = base.RefNotUniqueErr
+	NotFoundErr             = base.NotFoundErr
 )
 
 //
-// REST Client.
-type Client interface {
-	// Get a resource.
-	// The `resource` must be a pointer to a resource object.
-	// Returns: The HTTP code and error.
-	Get(resource interface{}, id string) (int, error)
-	// List a collection.
-	// The `list` must be a pointer to a slice of resource object.
-	// Returns: The HTTP code and error.
-	List(list interface{}) (int, error)
-}
+// Interfaces.
+type Client = base.Client
+type Finder = base.Finder
+type Param = base.Param
```

Comment:
Aggregated export of the base types.  Can be imported and used as `web.Client` etc.